content/webapp/pages/exhibition-guide.tsx

```diff
@@ -1,78 +1,105 @@
+      client,
+      id as string
+    );
 
-  // const client = createClient(context);
-  // const exhibitionGuideDocument = await fetchExhibitionGuide(client, id as string);
-  const exhibitionGuideDocument = true;
+    // return {
+    //   props: removeUndefinedProps({
+    //     exhibitionGuide: {
+    //       hierarchyExample: constructHierarchy(testComponentData),
+    //     },
+    //     // jsonLd, TODO Hierarchy
+    //     serverData,
+    //   }),
+    // };
+    if (exhibitionGuideDocument) {
+      const exhibitionGuide = transformExhibitionGuide(exhibitionGuideDocument);
+      const jsonLd = exhibitionGuideLd(exhibitionGuide);
 
-  if (exhibitionGuideDocument) {
-    // const exhibitionGuide = transformExhibitionGuide(exhibitionGuideDocument);
-    // const jsonLd = exhibitionGuideLd(exhibitionGuide);
-
-    return {
-      props: removeUndefinedProps({
-        exhibitionGuide: {
-          title: 'This is an exhibition guide',
-        },
-        // jsonLd, TODO
-        serverData,
-      }),
-    };
-  } else {
-    return { notFound: true };
-  }
-};
+      return {
+        props: removeUndefinedProps({
+          exhibitionGuide,
+          jsonLd,
+          serverData,
+        }),
+      };
+    } else {
+      return { notFound: true };
+    }
+  };
 
 const ExhibitionGuidesPage: FC<Props> = ({
   exhibitionGuide,
-  id,
-  type, // TODO keep types format in Prismic same as we do for Guides, Q. for PR
-  /* jsonLd */ // TODO
+  jsonLd,
+  // type, // TODO keep content-types format in Prismic same as we do for Guides, Q. for PR
 }) => {
-  const pathname = '/guides/exhibitions/'; // TODO /id/type
+  const pathname = `guides/exhibition/${exhibitionGuide.id}`; // TODO /id/content-type
   return (
     <PageLayout
-      title={'Exhibition Guide'} // TODO title
-      description={pageDescriptions.exhibitionGuides} // TODO
+      title={exhibitionGuide.title || ''}
+      description={pageDescriptions.exhibitionGuides}
       url={{ pathname: pathname }}
-      jsonLd={{ '@type': 'WebPage' }} // TODO, get jsonLd properly and what is the correct type
+      jsonLd={jsonLd}
       openGraphType={'website'}
-      siteSection={'collections'}
+      siteSection={'whats-on'}
       image={undefined} // TODO, linked doc promo image, e.g. Exhibition image
     >
-      <p>EXHIBITION GUIDE</p>
+      <p>Wellcome Collection Exhibition guide: {exhibitionGuide.title}</p>
+      <pre
+        style={{
+          maxWidth: '600px',
+          margin: '0 auto 24px',
+          fontSize: '14px',
+        }}
+      >
+        <code
+          style={{
+            display: 'block',
+            padding: '24px',
+            backgroundColor: '#EFE1AA',
+            color: '#000',
+            border: '4px solid #000',
+            borderRadius: '6px',
+          }}
+        >
+          <details>
+            <summary>exhibition-guide</summary>
+            {/* eslint-disable-next-line no-restricted-syntax */}
```

Comment:
Pleased to see the linting rule I added to stop us accidentally showing JSON on pages actually works!